en/docs/what-is-devant.md

```diff
@@ -1,18 +1,20 @@
 # What is Devant?
 
-Devant allows you to manage your own AI integration agents. Integrate any AI agent into the integrations you build in low-code and pro-code, and jump from siloed systems to intelligent digital experiences with Devant by WSO2—the AI iPaaS that your AI Agents can call “home”.
+Devant is a powerful IPaaS with first-class AI support. Integrate any AI agent into the integrations you build in low-code and pro-code, and move away from siloed systems to intelligent digital experiences with Devant by WSO2—the AI iPaaS that your AI Agents can call “home”.
 
 ## Key capabilities of Devant
 
-### Design for security, governance, and efficiency
-- **Domain-Driven Design**: Organize enterprise systems into domains, exposing managed APIs, events, and data through projects. Each project consists of components representing units of deployment.
-- **Application Security Design**: Leverage [cell-based architecture](https://github.com/wso2/reference-architecture/blob/master/reference-architecture-cell-based.md), where projects become governable units from development to deployment, ensuring managed access via APIs, events, and data.
-
 ### Efficient and effective development
-- **Code in Any Language**: Develop applications in your team's preferred programming language, harnessing existing skill sets to build powerful polyglot APIs, services, events, tasks, and more.
-- **Native Support for VS Code**: Seamlessly collaborate and manage code using VS Code in a feature-rich, visually-enhanced development environment.
+- **AI-Assisted Development**: Devant includes a developer friendly VSCode-based editor with an integrated AI copilot, which aids developers at every stage, from code generation to refactoring.
+- **Unified Low-Code and Pro-Code Development**: Allows seamless transitions between low-code and pro-code development, accommodating a diverse range of developer skills and preferences.
+- **Comprehensive Protocol and AI Support**: Built-in capabilities to integrate with third-party AI services, vector databases, and support for protocols such as TCP, UDP, FILE, Websub, and Websubhub.
 - **Git Repository Integration**: Effortlessly connect to Git-based platforms like GitHub, Bitbucket, or GitLab to develop components by linking an existing repository.
 
+### Design for security, governance, and developer experience
+- **Application Security Design**: Leverage [cell-based architecture](https://github.com/wso2/reference-architecture/blob/master/reference-architecture-cell-based.md), where projects become governable units from development to deployment, ensuring managed access via APIs, events, and data.
```

Comment:
Shouln't cell-based architecture should be the title of the point. We are tring to market cell-based architecture as well.